src/DependencyGraph/AddressMapping/AddressMapping.ts

```diff
@@ -6,6 +6,7 @@
 import {SimpleCellAddress} from '../../Cell'
 import {RawCellContent} from '../../CellContentParser'
 import {EmptyValue, InterpreterValue} from '../../interpreter/InterpreterValue'
+import {Maybe} from '../../Maybe'
```

Comment:
Let us do the cleanup of imports: once, automatically, on the develop, once all the branches are merged. Webstorm can do this, but this will touch basically all the files. If we do it here, it will be hell to merge with follow-up branches.

Reply:
> Or should the `getCell` type be updated?
> 
> ```diff
> - getCell(address: SimpleCellAddress): CellVertex | null
> + getCell(address: SimpleCellAddress): Maybe<CellVertex>
> ```

at one point yes. This is on my internal todo-list.